templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
+This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.
```

Comment:
reviewer group --> review team might sound better.
Please change Pycon to `PyCon India` everywhere with C capital


```suggestion
This email is in regard to the PyCon India 2020 review team. Pycon India is going virtual this year considering the current situations (LINK the announcement tweet). We (conference organizers) take this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and more accessible to our esteemed attendees from all over the world in ways more than one.
```

Reply:
Please add/modify any changes I suggested above into your text, as you like.